.gitignore

```diff
@@ -1,3 +1,8 @@
 gh-pages
 site
 **/node_modules
+.terraform
+*tfstate*
```

Comment:
where are we going to store the TF state?

Reply:
I've added instructions into README to store the state into an Azure storage container.